.github/workflows/private-package.yml

```diff
@@ -0,0 +1,47 @@
+name: private-package
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
+  pull_request: {}
+  workflow_dispatch: {}
+
+env:
+  GO_VERSION: '1.18'
+  KIND_VERSION: 'v0.14.0'
+
+jobs:
+  provider-upgrade-crossplane-stable:
```

Comment:
can we make this a more applicable name? e.g. `private-package-pull`?

Reply:
whoops! Yes absolutely :)